lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -35,11 +35,12 @@ public function register_routes() {
 			}
```

Comment:
The API has always supported the public query_vars of WP_Query, included a few others, and provided a filter to add support for additional vars.  There are some public query vars that are pretty pointless (example `withcomments`) but trying to normalize all these params sounds like a headache to me.